maas/maas_common.py

```diff
@@ -71,7 +72,7 @@ def get_cinder_client(previous_tries=0):
         return cinder
 
 try:
-    from glanceclient import Client as g_client
+    import glanceclient as g_client
```

Comment:
Little nit here, later on line 121 we're importing novaclient.client as nova_client, we should probably keep with that convention and call this glance_client.